collector/pkg/component/analyzer/cpuanalyzer/delete_tid.go

```diff
@@ -0,0 +1,76 @@
+package cpuanalyzer
+
+import (
+	"sync"
+	"time"
+)
+
+type tidDeleteQueue struct {
+	queueMutex sync.Mutex
+	queue      []deleteTid
+}
+
+var tidExpiredQueue *tidDeleteQueue
```

Comment:
Better make `tidExpiredQueue` a variable of `cpuanalyzer`, instead of as a package level one.

Reply:
ok